rskj-core/src/main/java/co/rsk/core/bc/BlockResult.java

```diff
@@ -91,26 +86,32 @@ public long getPaidFees() {
         //TODO Fix Trie hash for receipts - doesnt match cpp
         Trie receiptsTrie = new TrieImpl();
 
-        if (receipts == null || receipts.isEmpty())
+        if (receipts == null || receipts.isEmpty()) {
             return HashUtil.EMPTY_TRIE_HASH;
+        }
 
-        for (int i = 0; i < receipts.size(); i++)
+        for (int i = 0; i < receipts.size(); i++) {
             receiptsTrie = receiptsTrie.put(RLP.encodeInt(i), receipts.get(i).getEncoded());
+        }
 
         return receiptsTrie.getHash();
     }
 
     private static byte[] calculateLogsBloom(List<TransactionReceipt> receipts) {
         Bloom logBloom = new Bloom();
 
-        for (TransactionReceipt receipt : receipts) {
-            logBloom.or(receipt.getBloomFilter());
+        if (receipts != null) {
+            for (TransactionReceipt receipt : receipts) {
+                logBloom.or(receipt.getBloomFilter());
+            }
         }
 
         return logBloom.getData();
     }
 
-    public boolean getInterruptedExecution() {
-        return interruptedExecution;
+    private static class InterruptedExecutionBlockResult extends BlockResult {
+        public InterruptedExecutionBlockResult() {
+            super(null, null, null, 0, null);
```

Comment:
Could we use empty values instead of null? Am empty List and BigInteger.Zero would be nicer to handle so inside of the class we don't need to check we are receiving the valid values.

Reply:
Done https://github.com/rsksmart/rskj/pull/304/commits/19002c019392387e53dad0b31262ef99d6e4759c